app/migrations/Version20180425145459.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20180425145459 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('
+            CREATE TABLE adherent_email_subscription_history (
+                id BIGINT AUTO_INCREMENT NOT NULL, 
+                adherent_id INT UNSIGNED DEFAULT NULL, 
+                referent_tag_id INT UNSIGNED DEFAULT NULL, 
```

Comment:
why do we need to know the referent tag of adherent on subscription?

Reply:
We need that to filter by tag (which refers to area and more)